server/publications/selectedProduct.js

```diff
@@ -0,0 +1,3 @@
+Meteor.publish("selectedProduct", function(searchedCode){
+  return Products.find( {itemLookUpCode: searchedCode} );
```

Comment:
I apologized, should have made sure I understood the style guide better before assuming I understood what your last comment on this meant